layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,231 @@
+      <li>Find and donate to bail fund via the <a href="https://www.communityjusticeexchange.org/nbfn-directory">National Bail Fund Network</a>.</li>
+      <li><a href="https://www.facebook.com/reclaimtheblock/">Reclaim the Block</a> has published a <a href="https://docs.google.com/document/d/1yLWGTQIe3967hdc9RSxBq5s6KKZHe-3_mWp5oemd7OA/preview?pru=AAABcpUiX3k*Y6Q4I6UBtkH3lLz9GVLg0A">Google Doc of organizations to donate to.</a></li>
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+
+    <p>It would be wrong for us to ignore the unearned privilege that exists in Node.js: much of the project leadership is white, and a majority are men. We are and have been systematically complicit in perpetuating the issues that led to where we are now. We can't change that, but we can change how we proceed moving forward. We are dedicated to being a part of the solution by actively centering the Black community with our collective platforms to uplift Black voices, share projects founded and built by Black community members, and by financially supporting work from Black creators.</p>
+    <p>The labor to dismantle white supremacy does not fall on Black community; it falls on the rest of us. If there are Black folks looking for collaboration, amplification, mentorship, guidance, or any sort of support, we're here for you (see below).</p>
+
+ <p>Beginning with our upcoming Collaborator Summit event, and extending beyond, we will be offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If you are interested in this, please send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or direct message us at <a href="https://twitter.com/nodejs">@nodejs</a> on Twitter. We will post more information on how we will be proceeding with this effort as we finalize the details both for the period of the Collaborator summit and beyond that single event.</p>
+
+    <section id="we-are-here-for-you">
+      <p>Please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively, or via Twitter DMs to the following folks in the project:
+      <ul>
+        <li><a href='https://twitter.com/bitandbang'>Tierney Cyren</a></li>
+        <li><a href='https://twitter.com/joe_sepi'>Joe Sepi</a></li>
+        <li><a href='https://twitter.com/jasnell'>James M Snell</a></li>
+        <li><a href='https://twitter.com/MylesBorins'>Myles Borins</a></li>
+        <li><a href='https://twitter.com/mmarkini'>Matheus Marchini</a></li>
+        <li><a href='https://twitter.com/obensource'>Ben Michel</a></li>
+        <li><a href='https://twitter.com/codeekage'>Abraham Jr. Agiri</a></li>
```

Comment:
```suggestion
        <li><a href='https://twitter.com/codeekage'>Abraham Jr. Agiri</a></li>
        <li><a href='https://twitter.com/obensource'>Ben Michel</a></li>
        <li><a href='https://twitter.com/jasnell'>James M Snell</a></li>
        <li><a href='https://twitter.com/joe_sepi'>Joe Sepi</a></li>
        <li><a href='https://twitter.com/mmarkini'>Matheus Marchini</a></li>
        <li><a href='https://twitter.com/MylesBorins'>Myles Borins</a></li>
        <li><a href='https://twitter.com/bitandbang'>Tierney Cyren</a></li>
```

Alphabetize

Reply:
gonna go ahead and land this, I do not believe it is controversial and we should not have this list be first come first serve IMHO